kie-dmn-jpmml/src/main/java/org/kie/dmn/jpmml/DMNjPMMLInvocationEvaluator.java

```diff
@@ -89,7 +89,9 @@ public EvaluatorResult evaluate(DMNRuntimeEventManager eventManager, DMNResult d
                 result.put(outputFieldName, EvalHelper.coerceNumber(resultsRecord.getOrDefault(outputFieldName, null)));
             }
         }
-        return new EvaluatorResultImpl(result, ResultType.SUCCESS);
+
+        Object coercedResult = result.size() > 1 ? result : result.values().iterator().next();
+        return new EvaluatorResultImpl(coercedResult, ResultType.SUCCESS);
```

Comment:
Is this a part of the PR?

Reply:
From Kie `master` (merged instead of rebased).